config/tests/rechunk.yaml

```diff
@@ -0,0 +1,117 @@
+##########################################
+#
+#
+##########################################
+---
+type: config
+# Logging level can be one of:
+# OFF, ERROR, WARN, INFO, DEBUG, DEBUG2 (case insensitive)
+# Note DEBUG and DEBUG2 require a build with (-DCMAKE_BUILD_TYPE=Debug)
+log_level: info
+num_links: 4
+freq_array: [4,7,10,16]
+timesamples_per_packet: 2
+num_gpus: 1
+num_gpu_frames: 1
+cpu_affinity: [2,3,4,5,8,9,10,11]
+sizeof_float: 4
+sizeof_float16: 2
+sizeof_int: 4
+
+# td: 52
+# num_local_freq: 256
+# num_beams: 5000
+# tchunked: 260
+
+td: 1
+num_local_freq: 1
+num_beams: 1
+tchunked: 5
+
+frame_arrival_period: 0.001 * td
+
+buffer_depth: 4
+
+# data gen:
+num_frames: 20
+
+# Pool
+main_pool:
+    kotekan_metadata_pool: oneHotMetadata
+    num_metadata_objects: 15 * buffer_depth
+
+# Buffers
+host_input_buffer:
+    kotekan_buffer: standard
+    num_frames: buffer_depth
+    #frame_size: td * num_local_freq * num_beams * sizeof_float16
+    frame_size: td * num_local_freq * num_beams
+    metadata_pool: main_pool
+
+host_output_buffer:
+    kotekan_buffer: standard
+    num_frames: buffer_depth
+    #frame_size: tchunked * num_local_freq * num_beams * sizeof_float16
+    frame_size: tchunked * num_local_freq * num_beams
+    metadata_pool: main_pool
+
+gen_data:
+    kotekan_stage: testDataGen
+    #type: constf16
+    #value: 42.0
+    type: onehot
+    values: [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]
+    out_buf: host_input_buffer
+
+gpu:
+    log_level: debug
+    profiling: true
+    log_profiling: true
+    kernel_path: "../../lib/cuda/kernels/"
+    commands: &command_list
+    - name: cudaInputData
+      in_buf: host_input
+      gpu_mem: input
+    - name: cudaSyncInput
+    - name: cudaRechunk
+      gpu_mem_input: input
+      gpu_mem_output: output
+      # len_inner_input: td * sizeof_float16
+      # len_inner_output: tchunk * sizeof_float16
+      # len_outer: num_local_freq * num_beams
+      # len_inner_input: 104
+      # len_inner_output: 520
+      # len_outer: 1280000
+      len_inner_input: 1
+      len_inner_output: 5
+      len_outer: 1
+    - name: cudaSyncOutput
+    - name: cudaOutputData
+      gpu_mem: output
+      out_buf: host_output
+      in_buf: host_input # Metadata transfer from here
+    gpu_0:
+        kotekan_stage: cudaProcess
+        gpu_id: 0
+        commands: *command_list
+        in_buffers:
+            host_input: host_input_buffer
+        out_buffers:
+            host_output: host_output_buffer
+
+hexDump:
```

Comment:
This test only prints things, it doesn't test them, should we keep the config in the repo, or was this intended as a temporary debugging config?